### DISCUSSION TITLE
Change canvas view from another react component

### DISCUSSION THREAD

**Discussion by ameya730** [GrapesJS/grapesjs#4633] [Q&A]
Hi @artf ,

I am currently using grapejs in my application [using Reactjs]. However, the client wants the design to be as per his specifications.
This involves having the buttons to toggle between mobile & desktop view in a different react component.

Is there a way I can change the canvas view [toggle between mobile & desktop] from the other react component ?

Regards,
Ameya

**Comment by artf** [comment]
You can change the device programmatically https://grapesjs.com/docs/api/device_manager.html#select
